src/Workleap.Extensions.Mongo.Analyzers.Tests/IndexAttributeUsageAnalyzerTests.cs

```diff
@@ -75,22 +166,22 @@ public class PersonDocument : IMongoDocument { }
 
 public partial class MyWorker
 {
-    public void DoSomething()
+    public void DoSomething2()
```

Comment:
I know you didn't change the name of this tests, but at line 162 the name is wrong. It says there should be a diagnostic on each class when in fact there is only 1 diagnostic